care/emr/signals/charge_item_def/schedule.py

```
@@ -0,0 +1,61 @@
+            _OLD_REVISIT_CHARGE_ITEM_DEF_ID,
+            old_instance.revisit_charge_item_definition_id,
+        )
+        setattr(instance, _OLD_DELETED, old_instance.deleted)
+    else:
+        setattr(instance, _OLD_CHARGE_ITEM_DEF_ID, None)
+        setattr(instance, _OLD_REVISIT_CHARGE_ITEM_DEF_ID, None)
+        setattr(instance, _OLD_DELETED, False)
+
+
+@receiver(post_save, sender=Schedule)
+def update_separately_billable_schedule(sender, instance, created, **kwargs):
```

Ruff (0.14.13)

Unused function argument: `sender`

(ARG001)